import_activity_events.py

```diff
@@ -173,11 +176,11 @@ def import_events(force_reload=False):
         db.run("COMMIT TRANSACTION")
 
 
-def fixup_event_data(b, week):
+def fixup_event_data(b, day):
     """Download a data file, remove invalid lines, and re-upload"""
-    orig_key = b.get_key(urlparse.urlparse(EVENTS_FILE_URL.format(week=week)).path[1:])
+    orig_key = b.get_key(urlparse.urlparse(EVENTS_FILE_URL.format(day=day)).path[1:])
     assert orig_key is not None
-    fixed_key = b.new_key(urlparse.urlparse(EVENTS_FILE_FIXED_URL.format(week=week)).path[1:])
+    fixed_key = b.new_key(urlparse.urlparse(EVENTS_FILE_FIXED_URL.format(day=day)).path[1:])
```

Comment:
> Should we bite the bullet and migrate to a new table (say "activity_events") for the new 100%-sample-rate data, and keep it separate from the old sampled data?

Absolutely. I should have mentioned, I haven't touched the activity event script since our 1:1 the other day, so none of that stuff is taken into account yet. For the record, my recollection is single table, then make sure we're happy with that schema, then ask Wes about the possibility of retro-exporting old data at the new frequency/sample rate and with an empty device id.

I'll make sure I bring this script up to date today.